### PR TITLE
DIV-4485: changing landing page title for amended cases

### PR DESCRIPTION
**Pull request by jvictorcf**
# Description

- Updating title for landing pages for respondent and co-respondent.
- Enable prod deployments 
## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Unit tests and manual testing with mocks

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: mocks/services/case-orchestration/retrieve-aos-case/mock-co-respondent-amended-case.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+      "contactInfo": {
+        "emailAddress": "user@email.com"
+      },
+      "aos": {
+        "received": "Yes",
+        "letterHolderId": "755791",
+        "dateReceived": "2019-02-22"
+      }
+    },
+    "d8": [
```
Comment (alexwakeman) [inline mocks/services/case-orchestration/retrieve-aos-case/mock-co-respondent-amended-case.json]: Hey @jvictorcf - we're not sending back any documents to the FE apps any more when doing an Amend workflow - so we don't need these entries.

Comment (jvictorcf) [reply mocks/services/case-orchestration/retrieve-aos-case/mock-co-respondent-amended-case.json]: Hey @alexwakeman, good spot.
I'm happy to remove it. However, just for my own clarification, in the RFE, couldn't the petitioner have added these documents to the amended case again?

Comment (alexwakeman) [reply mocks/services/case-orchestration/retrieve-aos-case/mock-co-respondent-amended-case.json]: Actually yes you're right my bad... this is Respondent app, - ignore me

**Comment by hosainnet** [comment]
This is now signed off @jvictorcf FYI

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -59,13 +59,5 @@ withPipeline("nodejs", product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'functional-output/**/*'
   }
 
-  before('buildinfra:prod') {
```
Comment (qzhou-hmcts) [inline Jenkinsfile_CNP]: Enable prod deployments

**Pull request merged**